text/0002-sarif-result-schema.md

```diff
@@ -0,0 +1,1939 @@
+  }
+
+  /**
+  * A component, such as a plug-in or the driver, of the analysis tool that was run.
+  */
+  export interface ToolComponent {
+
+      /**
+      * The kinds of data contained in this object.
+      */
+      contents?: ToolComponent.contents[];
+
+      /**
+      * A comprehensive description of the tool component.
+      */
+      fullDescription?: MultiformatMessageString;
+
+      /**
+      * The name of the tool component along with its version and any other useful identifying information, such as its
+      * locale.
+      */
+      fullName?: string;
+
+      /**
+      * A unique identifer for the tool component in the form of a GUID.
+      */
+      guid?: string;
+
+      /**
+      * The minimum value of localizedDataSemanticVersion required in translations consumed by this component; used by
+      * components that consume translations.
+      */
+      minimumRequiredLocalizedDataSemanticVersion?: string;
+
+      /**
+      * The name of the tool component.
+      */
+      name: string;
+
+      /**
+      * An array of reportingDescriptor objects relevant to the notifications related to the configuration and runtime
+      * execution of the tool component.
+      */
+      notifications?: ReportingDescriptor[];
+
+      /**
+      * The organization or company that produced the tool component.
+      */
+      organization?: string;
+
+      /**
+      * A product suite to which the tool component belongs.
+      */
+      product?: string;
+
+      /**
+      * A localizable string containing the name of the suite of products to which the tool component belongs.
+      */
+      productSuite?: string;
+
+      /**
+      * A string specifying the UTC date (and optionally, the time) of the component's release.
+      */
+      releaseDateUtc?: string;
+
+      /**
+      * An array of reportingDescriptor objects relevant to the analysis performed by the tool component.
+      */
+      rules?: ReportingDescriptor[];
+
+      /**
+      * The tool component version in the format specified by Semantic Versioning 2.0.
+      */
+      semanticVersion?: string;
+
+      /**
+      * A brief description of the tool component.
+      */
+      shortDescription?: MultiformatMessageString;
+
+      /**
+      * The tool component version, in whatever format the component natively provides.
+      */
+      version?: string;
+
+      /**
+      * Key/value pairs that provide additional information about the tool component.
+      */
+      properties?: PropertyBag;
+  }
+
+  /**
+  * The runtime environment of the analysis tool run.
+  */
+  export interface Invocation {
+      /**
+      * An array of strings, containing in order the command line arguments passed to the tool from the operating
+      * system.
+      *
+      * CHECKUP: args go heree 
+      */
+      arguments?: string[];
+
+      /**
+      * The command line used to invoke the tool.
+      */
+      commandLine?: string;
```

Comment:
Is this the full command line string that's used to invoke the tool?

Reply:
Yes